.travis.yml

```diff
@@ -7,9 +7,18 @@ branches:
     - master
     - dev
 
+before_install:
+    - sudo add-apt-repository -y ppa:snaipewastaken/ppa
+    - sudo apt-get update -q
+    - sudo apt-get install criterion-dev -y
+
 jobs:
   include:
     - stage: build
-      script: echo Travis # dummy command until we have make files
+      script: make clean all
     - stage: test
-      script: echo CI # dummy command until we have make files
+      script:
+          - make clean all
```

Comment:
Here we definitely *don't* want to run `make clean`, nor `make all`. You should use artifacts from the `build` stage to avoid compiling your code an extra time unnecessarily.

Reply:
Sorry just seeing this now. This is on me since I didn't understand (and still don't really know) how to make artifacts for a .travis.yml file. I can create a separate issue for this to figure it out. @mpcsh @dengyijia